lib/diagnostic.rb

```diff
@@ -105,16 +122,23 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+    puts 'ROAR!'
+  end
+end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
-##
+# # Using mixins allows you to include some methods when creating a new class that is similar or behaves in the
```

Comment:
This is true, but it doesn't explain why composition is advantageous over inheritance.  The primary reason is that a class can only inherit from one other class.  Using composition, any number of modules can be included in a class.